components/Filter.js

```diff
@@ -1,7 +1,14 @@
 import React from "react";
 
-const Filter = () => {
-	return <div>Filter</div>;
+const Filter = ({ topic }) => {
+	return (
+		<div className='w-[270px] h-[55px] flex justify-between bg-black rounded-full'>
+			<div className='text-white w-9 h-9 self-center ml-5 bg-[#ff5370] rounded-full'></div>
```

Comment:
Avoid `w-9`, `ml-5` and `h-9` as they also cause unpredictable behavior with various screen sizes, use percentages instead